jep/1/README.adoc

```diff
@@ -19,7 +19,7 @@ endif::[]
 | Title
 | Jenkins Enhancement Proposal Format
 
-| Author
+| Sponsor
```

Comment:
"Sponsor" is better than "Author" for sure. It may be also a bit ambiguous because it's also may be considered as a "person who pays the bill". OTOH other projects use "sponsor" for similar purposes, so I am fine with that.

Reply:
@oleg-nenashev - I agree, it is imperfect.  But it avoids "Owner" and "Author".  It is sponsor more in the sense of "Someone who vouches for and guides".   

RESOLVED.